extensions/amp-viewer-integration/0.1/messaging.js

```diff
@@ -42,6 +41,22 @@ const MessageType = {
  */
 export let Message;
 
+export class WindowPortEmulator {
```

Comment:
Please add jsdoc and mention that this de-facto implements `MessagePort` and provide a link to mdn.